### PR TITLE
[WOR-545] Changing cloning instructions on storage container.

### PR DESCRIPTION
**Pull request by cahrens**
Ticket: https://broadworkbench.atlassian.net/browse/WOR-545

---

**PR checklist**

- [x] Include the JIRA issue number in the PR description and title
- [ ] Make sure Swagger is updated if API changes
  - [ ] **...and Orchestration's Swagger too!**
- [ ] If you changed anything in `model/`, then you should [publish a new official `rawls-model`](https://github.com/broadinstitute/rawls/blob/develop/README.md#publish-rawls-model) and update `rawls-model` in [Orchestration's dependencies](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/project/Dependencies.scala).
- [ ] Get two thumbsworth of PR review
- [ ] Verify all tests go green, including CI tests
- [ ] **Squash commits and merge** to develop (branches are automatically deleted after merging)
- [ ] Inform other teams of any substantial changes via Slack and/or email

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -39,6 +39,7 @@ jobs:
 
       - name: Secrets check
         run: |
+          export TERM=dumb
```
Comment (cahrens) [inline .github/workflows/test.yml]: I really doubt this is actually helping work around the "say not installed" issue. But I also don't think there is any real harm in having it here. [Slack thread](https://broadinstitute.slack.com/archives/C03BCARP494/p1665766224753949) with context.

**Comment by cahrens** [comment]
Swatamation problem is fiab-start, argocd issue

````
09:54:53  cmd=5d4056
09:54:53 1:54PM ERR error executing job error="Command \"ARGOCD_SERVER=ap-argocd.dsp-devops.broadinstitute.org ARGOCD_AUTH_TOKEN=****** argocd --header Proxy-Authorization: Bearer ****** --grpc-web app wait workspacemanager-jenkins-swat-72031 --timeout 600 --health\" exited with status 20:\ntime=\"2022-10-19T13:54:53Z\" level=fatal msg=\"application 'workspacemanager-jenkins-swat-72031' health state has transitioned from Progressing to Degraded\"\n" item=workspacemanager-jenkins-swat-72031 wid=worker-5
09:54:53 1:54PM DBG job execution finished in 10m35.86976165s item=workspacemanager-jenkins-swat-72031 wid=worker-5
09:54:53 1:54PM DBG queue empty, returning wid=worker-5

**Comment by cahrens** [comment]
jenkins retest

**Review by marctalbott — APPROVED** [review]

**Comment by cahrens** [comment]
jenkins retest

**Review by blakery — APPROVED** [review]

**Pull request merged**